### ISSUE TITLE
Tensor Developer Summit 2020

### ISSUE THREAD

**Issue by alimanfoo** [zarr-developers/community#26]
There will be a Tensor Developer Summit in March at UC Berkeley: https://xd-con.org/tensor-2020/

**Comment by alimanfoo** [comment]
Unfortunately I won't be able to attend, but Stéfan van der Walt (local organiser) reached out to say that anyone from @zarr-developers/core-devs would be welcome.

**Comment by jakirkham** [comment]
Yeah I saw this as well. Unfortunately have too much going on around that time to go. Though agree it would be nice if someone could attend.

**Comment by jrbourbeau** [comment]
I'll be out of town at that time unfortunately. Looks like an awesome event

**Comment by mzjp2** [comment]
I can tentatively attend, I've been pushing Zarr heavily at work (QuantumBlack) and I could probably convince them to send me along!

**Comment by alimanfoo** [comment]
> I can tentatively attend, I've been pushing Zarr heavily at work (QuantumBlack) and I could probably convince them to send me along!

:+1: 

P.S. QuantumBlack definitely wins the prize for the sexiest website, very smart :smile:

**Comment by rabernat** [comment]
I am really pleased that someone from zarr (@mzjp2) can attend this important event.

**Comment by ZainPatelQB** [comment]
Will be taking lots of notes and write up the things relevant to Zarr and share it back. Looking forward to it! 🤞

**Comment by mzjp2** [comment]
Will be taking lots of notes and write up the things relevant to Zarr and share it back. Looking forward to it! 🤞

**Comment by ryan-williams** [comment]
I applied to attend (and speak, if appropriate), I think before the deadline last week. I don't think I've heard back yet but am excited to try to go as well

**Comment by jakirkham** [comment]
Thanks for doing that Ryan! 😄

**Comment by ryan-williams** [comment]
Update: I heard back that I'm on the list for the event as well, so I'll be there!

**Comment by ZainPatelQB** [comment]
Same, see you there @ryan-williams! :D

**Comment by mzjp2** [comment]
Same, see you there @ryan-williams! 😀

**Comment by jakirkham** [comment]
This is great! Thank you both for representing Zarr there! 😄

**Comment by jakirkham** [comment]
Is the summit still happening? Asking as I know some events have been canceled due to the Coronavirus.

**Comment by mzjp2** [comment]
Nope, cancelled and a tentative proposal to reconsider a date in ~September, subject to change.

**Comment by jakirkham** [comment]
> Nope, cancelled and a tentative proposal to reconsider a date in ~September, subject to change.

Thanks for the update Zain! I'm guessing you already saw this @ryan-williams 😉
